### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by anaPerezGhiglia**
### Changes
- add translations  for new `SuccessRateLines` labels introduced in #2336 
    **Spanish**
    image
   **French**
    image

- add `es` and `fr` translations for missing labels of survey overview success rate cockpit

| Lang | Before | After |
|------|--------|------|
| Spanish | image |  image |
| French | image | image |

### Considerations
 - I got the translations by using Google Translator & common sense for Spanish. @ysbaddaden could you take a look at the French ones?

### CODE REVIEW

File: locales/translations/es.json [hunk 1]
```diff
@@ -58,6 +58,8 @@
     "activities": "registros",
     "activity": "registro",
     "Activity": "Actividad",
+    "Actual success rate": "Tasa de éxito actual",
```
Comment (matiasgarciaisaia) [inline locales/translations/es.json:61]:
```suggestion
    "Actual success rate": "Tasa de éxito real",
```

Other options would be `efectiva` or `medida`, but `real` is OK. Spanish's `actual` is English's `current`.

File: locales/translations/es.json [hunk 2]
```diff
@@ -58,6 +58,8 @@
...
+    "Actual success rate value throughout the survey's life": "Valor real de la tasa de éxito a lo largo de la vida de la encuesta",
```
Comment (matiasgarciaisaia) [inline locales/translations/es.json:62]:
```suggestion
    "Actual success rate value throughout the survey's life": "Tasa de éxito real a lo largo de la vida de la encuesta",
```

**Review by matiasgarciaisaia — COMMENTED** [review]
Can't comment on French 🙃

File: locales/translations/fr.json [hunk 3]
```diff
@@ -58,6 +58,8 @@
     "activities": "activités",
     "activity": "activité",
     "Activity": "Activité",
+    "Actual success rate": "Taux de réussite réel",
+    "Actual success rate value throughout the survey's life": "Valeur réelle du taux de réussite tout au long de la durée de vie de l'enquête",
```
Comment (ysbaddaden) [inline locales/translations/fr.json:62]: Hey, IA translators ain't bad :+1:
I'd just simplify this sentence a little bit:

```suggestion
    "Actual success rate value throughout the survey's life": "Taux de réussite réel sur toute la durée de l'enquête",
```

**Review by ysbaddaden — APPROVED** [review]

**Pull request merged**